capstone-backend/src/main/java/com/google/sps/servlets/Utility.java

```diff
@@ -1,32 +1,41 @@
 package com.google.sps.servlets;
```

Comment:
Similarly here, all these changes are from other PRs. Try having this merge into `master` instead.